### ISSUE TITLE
isModified is not a function

### ISSUE THREAD

**Issue by davidsavoie1** [jagi/meteor-astronomy#634]
I'm sorry if it's obvious, but I can't find what's wrong here with my class named Service.
```
const servClass = new Service();
console.log('Created a new Service class instance.', servClass);
console.log('isModified', servClass.isModified());
```
If I execute this code, I get this error:
```
Uncaught TypeError: servClass.isModified is not a function
```
If I inspect my class object printed to the console, I do see in the class `__proto__` some functions as `raw`, `validate`, `set`, etc. but I don't see functions such as `isModified`. 

From my understanding of [the documentation](http://jagi.github.io/meteor-astronomy/#setting-and-getting-values), I would have thought that all class instance objects would come with these functions... no?

What am I missing?

**Comment by lukejagodzinski** [comment]
It's only true for classes that have collection attached to it, so that's why you don't see this method.

**Comment by davidsavoie1** [comment]
Aaaahh! That explains a lot! Thanks for the info! Is it indicated somewhere in the docs? I haven't seent anything about it...

**Comment by lukejagodzinski** [comment]
No, I think documentation doesn't mention it. It would be great if you could make a PR and add proper info.

**Comment by davidsavoie1** [comment]
I've never done that before, I'll see what I can do to contribute!

**Comment by lukejagodzinski** [comment]
I can help you with that :) it's not difficult. There is the [`gh-pages`](https://github.com/jagi/meteor-astronomy/tree/gh-pages) branch in which you have all the documentation. In [this](https://github.com/jagi/meteor-astronomy/blob/gh-pages/_includes/v2/key_concepts/setting_and_getting_values.md) file you have a description of the `isModified` method. So you can just modify this file and create PR from the level of GitHub. Once, I accept this PR it will be automatically pushed to the docs.
